libs/libsigar/Makefile

```diff
@@ -0,0 +1,43 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=libsigar
+PKG_VERSION:=1.6.4.1
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO:=git
```

Comment:
Given that PKG_SOURCE is missing, what kind of tarball does this default to? xz?

Reply:
It's xz